docs/en/serial-vault.md

```diff
@@ -0,0 +1,415 @@
+
+1. Login to the Serial Vault as the Brand SSO account.
+1. Select Models.
+
+
+
+![image][serial-vault-image2]
+1. Click the Signing Keys + button.
+
+
+
+![image][serial-vault-image3]
+
+
+
+![image][serial-vault-image4]
+
+1. Enter account-id of the Brand SSO account as the Signing Authority.
+1. Under Signing Key, click Browse, then find and select the previously
+created ASCII-armored serial key file.. 
+1. Click Save. 
+The list of models and signing keys displays, including this new one.
+
+## Creating a model
+
+You need to create a model. This corresponds to the model previously used when
+building an image. (Every system has a model.) At runtime, the system sends
+the model to the Serial Vault, which uses it to authenticate the system before
+generating a serial assertion.
+
+The model contains key data that must exactly match data in the model in the
+deployed system, including:
+
+ - brand-id: this is the account-id of the Brand SSO account
+ - model name
+
+Also, the system itself must have a gadget snap whose prepare-device hook
+defines required snapd variables. See “Gadget Snap” section below for details
+on these variables. 
+
+Create a model as follows:
+
+1. Click Models.
+On first use, you have no models.
+1. Click the “+” button to the right of Models:
+
+
+![image][serial-vault-image5]
+
+
+![image][serial-vault-image6]
+
+1. Enter the account-id of your Brand SSO account into the Brand field.
+1. Enter the model name (from your system’s model assertion) into the Model
+field.
+1. The API Key must match the setting in your gadget snap prepare-device hook
+(see Gadget Snap section above). If you leave this field empty, an API key is
+generated and  you must use this new value in your gadget snap prepare-device
+hook script. If your gadget snap prepare-device hook already defines the API
+key, you must copy it from the hook into this field. 
+1. Select the Serial Assertion Key corresponding to the serial key you
+previously uploaded.
+1. In System-User Assertion Key, select a key you previously uploaded. This
+can be the serial key, or another one.
+1. Click Save.
+
+This model is now active. The Serial Vault responds to requests from systems
+that match with a signed serial assertion that the system then uses to access
+a secure Brand Store. 
+
+The Models page now lists your model:
+
+![image][serial-vault-image7]
+
+You can edit an existing model by pressing a model’s pencil icon button.
+
+## Checking the Signing Log
+
+You can review the Signing Log to check whether serial assertions have been
+signed and issued.
+
+The Signing Log displays the list of serial numbers and device keys that have
+been used to sign serial assertions. You can also filter the log. 
+
+![image][serial-vault-image8]
+
+## Creating system-user assertions
+
+A system-user assertion allows you to create a user on an unmanaged Core
+system. The system-user assertion can be seeded into an image, or it can be
```

Comment:
"seeded" is a domain-specific term in my opinion and I think it'd be much more clear to say "pre-populated" instead, unless we can define above what seeding means.

Reply:
We also have a core doc on preseeding classic images though.

Reply:
Is that doc published already? We should link to that, or if not, take my suggestion and switch words.